deploy/docker-compose/server-rvi-client.yml

```diff
@@ -0,0 +1,97 @@
+version: '2'
+services:
+  
+  mysql:
+    image: advancedtelematic/mariadb
+    ports:
+      - '3306:3306'
+    expose:
+      - '3306'
+    environment:
+      MYSQL_ROOT_PASSWORD: "sota-test"
+      MYSQL_USER: "sota"
+      MYSQL_PASSWORD: "s0ta"
+      MYSQL_DATABASES: "sota_resolver sota_resolver_test sota_core sota_core_test"
+
+
+  rvi_server:
+    image: advancedtelematic/rvi
+    ports:
+      - 8801:8801
+      - 8807:8807
+    depends_on:
+      - mysql
+    stdin_open: true
+    tty: true
+    command: server
+
+  resolver:
+    image: advancedtelematic/sota-resolver
+    ports:
+      - 8081:8081
+    expose:
+      - 8081
+    depends_on:
+      - mysql
+      - rvi_server
```

Comment:
This is just a hack. The whole thing fails if core or resolver start running migrations before MariaDB is fully up and running. `depends_on` just means "don't start this container until all of its dependencies have started"; it's not synonymous with "links" from docker-compose v.1.